lambdapi.opam

```diff
@@ -26,7 +26,7 @@ depends: [
   "bindlib"      { >= "5.0.0" }
   "earley"       { >= "2.0.0" }
   "timed"        { >= "1.0"   }
-
+  "why3"
```

Comment:
Add version constraint.

Reply:
Added in 212de9192a6ec6e842b1e4e69929014ccfd1a91d